analytics_data_api/management/commands/generate_fake_course_data.py

```diff
@@ -161,7 +163,7 @@ def generate_weekly_data(self, course_id, start_date, end_date):
 
         logger.info("Generating new weekly course activity data...")
 
-        progress = tqdm(total=math.ceil((end_date - start).days / 7.0) + 1)
+        progress = tqdm(total=math.ceil((end_date - start).days / 7.0))
```

Comment:
This is unrelated to the scope of this PR, but I noticed a off-by-one error in the progress bar here while I was working on this code.